### PR TITLE
Add messageburst and messagelimit configuration

### PR DESCRIPTION
**Pull request by bastiandg**
This PR
- introduces messagelimit and messageburst as config parameters
- fixes [shellcheck](https://github.com/koalaman/shellcheck/)  issues in the entrypoint
- sorts the config values in the README

Sadly the messageburst and messagelimit are not well documented in mumble. it wasn't even mentioned in the 1.3.0 release. So a link to the corresponding PR was added.

### CODE REVIEW

File: mumble-server/docker-entrypoint.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ ICEFILE="/etc/murmur/ice.ini"
 WELCOMEFILE="/data/welcometext"
 
 setVal() {
-    if [ ${1} ] && [ "${2}" ]; then
+    if [ -n "${1}" ] && [ -n "${2}" ]; then
```
Comment (sudoforge) [inline mumble-server/docker-entrypoint.sh:9]: Strictly speaking, this isn't necessary, as:

* `$1` is never more than a single string matching `[a-zA-Z]+`
* `$2` is already quoted and thus fails the check if no value is set

**Comment by bastiandg** [comment]
Thank you for your suggestions. I'm happy to do what I can.

>     * Separate out the configuration property sorting into its own commit
done ✅
 
>     * Separate out each shellcheck violation, grouped by the notice number, into its own commit
done ✅

>     * Add some validation around `messagelimit` and `messageburst` to ensure the values are always >= 1

I'm conflicted on this. I understand your concern. But there is currently no parameter validation in the entrypoint. This is solely left to mumble itself to confirm them. Which is a very sensible approach, as the validation logic doesn't have to be implemented twice. In fact mumble already takes care of validating negative values and corrects them:

`messageburst`: [murmur/Server.cpp#L493](https://github.com/mumble-voip/mumble/blob/c84f0a5ebf337fa28ec37475537a584ad9dfaed3/src/murmur/Server.cpp#L493)
`messagelimit`: [murmur/Server.cpp#L489](https://github.com/mumble-voip/mumble/blob/c84f0a5ebf337fa28ec37475537a584ad9dfaed3/src/murmur/Server.cpp#L489)

File: mumble-server/docker-entrypoint.sh (hunk 1 above)
Comment (bastiandg) [reply mumble-server/docker-entrypoint.sh:9]: Yes you are right. I would argue though, that using `-n` is both more readable and more robust. So yes the condition could technically be shortened to 
```bash
    if [ $1$2 ] ; then
```
and still work.

But opting for the unambiguous, robust and more readable solution, is the way to go here.

Comment (sudoforge) [reply mumble-server/docker-entrypoint.sh:9]: Valid on all points, and:

> But opting for the unambiguous, robust and more readable solution, is the way to go here.

You're absolutely correct!

File: mumble-server/README.md [hunk 2]
```diff
@@ -237,5 +239,6 @@ Numbered tags follow the pattern:
 [mdoc-suggestPushToTalk]: https://wiki.mumble.info/wiki/Murmur.ini#suggestPushToTalk
 [mdoc-sslcertkey]: https://wiki.mumble.info/wiki/Murmur.ini#sslCert_and_sslKey
 [mdoc-sslCiphers]: https://wiki.mumble.info/wiki/Murmur.ini#sslCiphers
+[mgit-messagelimit]: https://github.com/mumble-voip/mumble/pull/3517
```
Comment (sudoforge) [inline mumble-server/README.md:242]: I don't _like_ this, but given that it's not included in the wiki and linking to `scripts/murmur.ini` in the upstream project repository doesn't make sense, it's the best we've got at the moment -- I'll update the wiki or at least prod the devs to.

Comment (bastiandg) [reply mumble-server/README.md:242]: Thanks for following up on this.

**Pull request merged**